pkgs/os-specific/darwin/xcode/default.nix

```diff
@@ -14,7 +14,9 @@ let requireXcode = version: sha256:
               rm -rf ${xip}
               pbzx -n Content | cpio -i
               rm Content Metadata
+              rcodesign verify Xcode.app/Contents/MacOS/Xcode
```

Comment:
Like the tools above in the linux case, `rcodesign` is bundled with nixpkgs, but I'm not sure how to leverage that here.

Referencing the derivations (e.g. `"${rcodesign}/result/bin/rcodesign"`) would allow copy-pasting the lines but force the builds ahead of time even if the user decided not to take this step.